backend/constants.js

```diff
@@ -523,7 +523,76 @@
 
 	define("PROJECT_DEFAULT_ID", "default");
 
+	// risks
+	define("RISK_FILTERS",{
+		"ids": {
+			"fieldName": "_id" ,
+			"type": "UUID"
```

Comment:
Maybe these types can be defined in constants as well?

Reply:
Yes, could do it